robot_base_frame: $(arg robot)/base_link
robot_radius: 0.    # to override with planner-specific launch 
transform_tolerance: 0.5

plugins: 
- {name: static_layer, type: "costmap_2d::StaticLayer"}
- {name: inflation_layer, type: 'costmap_2d::InflationLayer'}

static_layer:
  map_topic: /map
  track_unknown_space: false
